tools/deploy.sh

```diff
@@ -26,9 +26,7 @@ fi
 
 cp beestation.dmb beestation.rsc $1/
 cp -r _maps/* $1/_maps/
-cp -r icons/obj/food/* $1/icons/obj/food/
-cp -r icons/runtime/* $1/icons/runtime/
-cp -r sound/runtime/* $1/sound/runtime/
+cp -r icons/* $1/icons/
```

Comment:
Do we need to copy over all the icons? It'd be faster if we could limit it to a small sample set, but if that's not feasible then we can work around it.

Reply:
No, it's not feasible as any of the icons could theoretically be used in a spritesheet.

Reply:
Ugh. Alright